### DISCUSSION TITLE
Live Update in TexStudio (Live Update)

### DISCUSSION THREAD

**Discussion by The-Cyber-Thinkerer** [texstudio-org/texstudio#3284] [Q&A]
I am new to this software and have been loving it a lot so far. I  was wondering if there is a way to make "Auto-recompile document on changes" faster and without a bit lag. In other words, i would like to achieve  something similar to Overleaf's live update feature in the sense that whenever i type in the editor, it instantaneously updates in the viewer. I hope that my question/request is clear! I know that it might not be ideal for long or big documents due to the number of lines of codes needed to be compiled, but I'm just working on small pieces of document and would like to find a way to achieve it ! Thank you

**Comment by octaeder** [comment]
s. #3137
use [preview](https://texstudio-org.github.io/viewing.html#preview)

**Reply by The-Cyber-Thinkerer** [reply]
Thank you for the reply! But I don’t mean the Inline Preview of my typed code in the editor, I mean the compiled document in a separate window in pdf format (or any other format). I would like to see a live preview of the whole document compiled as i edit it in the editor.

**Comment by sunderme** [comment]
there is also options/internal pdf preview/auto-recompile document on changes
